02-wrangling.Rmd

```diff
@@ -146,7 +159,8 @@ lang_wide
 ```
 
 
-What is wrong with our untidy format above? From a data analysis perspective, this format is not ideal because, in this format, the outcome of the variable *region* (Toronto, Montreal, Vancouver, Calgary and Edmonton) is stored as column names. Thus it is not easily accessible for the data analysis functions we will want to apply to our data set. Additionally, the values of the *mother tongue* variable are spread across multiple columns. This will prevent us from doing any desired visualization or statistical tasks until we somehow combine them into one column. For instance, suppose we want to know which languages had the highest number of Canadians reporting it as their mother tongue among all five regions? This question would be very difficult to answer with the data in its current format. It would be much easier to answer if we tidy our data first. 
+
+What is wrong with our untidy format above? From a data analysis perspective, this format is not ideal because, in this format, the outcome of the variable *region* (Toronto, Montreal, Vancouver, Calgary and Edmonton) is stored as column names. Thus it is not easily accessible for the data analysis functions we will want to apply to our data set. Additionally, the values of the *mother tongue* variable are spread across multiple columns, which will prevent us from doing any desired visualization or statistical tasks until we combine them into one column. For instance, suppose we want to know which languages had the highest number of Canadians reporting it as their mother tongue among all five regions? This question would be tough to answer with the data in its current format. It would be much easier to answer if we tidy our data first. 
```

Comment:
I think a graphic of the wide data, that is not tidy, and then the tidied data would be nice here, before we dig into the code to tidy it would help learners see what they need to do, before they have to rationalize what the code does. I think this is one of the more complicated bits of wrangling in the book, so extra graphics might be helpful.

I wonder if a code graphic here also would be nice? Like the super awesome one you make for ggplot?

Reply:
Good point! added 4 new graphics (for each of the pivot_* sections: added a general example that is simpler and then added another graphic which summarized how we reshaped the language data specifically). I really struggled to try to make them clean and as clear as possible so comments are welcome!